app/Http/Controllers/registrationController.php

```diff
@@ -3,11 +3,82 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\Models\Users;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Redis;
 
 class registrationController extends Controller
 {
-    public function index()
+    public function registerShow()
     {
         return view('form');
     }
+    public function loginShow()
+    {
+        return view('login');
+    }
+    public function register(Request $request)
+    {
+        $request->validate(
+            [
+                'name' => 'required',
+                'email' => 'required|email',
+                'phone' => 'required',
+                'pass' => 'required | min:5 ',
+            ]
+        );
+        $user=new Users;
+        $user->name=$request->name;
+        $user->phone=$request->phone; 
+        $user->email=$request->email;
+        $user->pass=$request->pass;
+        $user->save();
```

Comment:
Can make a separate request page for validation.
Purpose: We should always aim to keep the controller cleaner with least lines of code possible.
Reference link:https://laravel.com/docs/9.x/validation#creating-form-requests
User::create($req->validated());